CONTRIBUTING.md

```diff
@@ -23,6 +31,22 @@ Once done, you're all set to make your first contribution!
 
 Please take a look at the current open issues to see if any of them interest you. If you are unsure how to get started please take a look at the README.
 
+## Committing
+
+Please take care in using good commit messages as they're useful for debugging, reviewing code, and generally just shows care and quality. [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) provides a good guideline. At a minimum,
```

Comment:
While this is true in general, it's less important when grouping all commits into PRs that will be squashed when merged and the branches subsequently deleted.  But I guess we should still keep this section, if only for the `[ci skip]` instruction, which is useful.  (I always forget about that!)